apps/server/cmd/server/main.go

```
@@ -42,24 +64,21 @@ func configuredAddress() string {
-		IdleTimeout:       serverIdleTimeout,
+func newHTTPServer(address string, handlers ...http.Handler) *http.Server {
+	handler := httpapi.NewRouter()
+	if len(handlers) > 0 && handlers[0] != nil {
+		handler = handlers[0]
 	}
+	return &http.Server{Addr: address, Handler: handler, ReadHeaderTimeout: 5 * time.Second, ReadTimeout: serverReadTimeout, IdleTimeout: serverIdleTimeout}
 }
 
-func run(ctx context.Context, address string) error {
+func run(ctx context.Context, address string, handlers ...http.Handler) error {
 	listener, err := net.Listen("tcp", address)
```

golangci-lint (2.13.2)

[error] net.Listen must not be called. use (*net.ListenConfig).Listen

(noctx)